mod-source-record-storage-server/src/main/java/org/folio/services/RecordService.java

```diff
@@ -189,4 +191,13 @@
    */
   Future<Record> updateSourceRecord(ParsedRecordDto parsedRecordDto, String snapshotId, String tenantId);
 
+  /**
+   * Find marc bib ids by incoming arrays from SRM and exclude all valid marc bib and return only marc bib ids,
+   * which does not exists in the system
+   *
+   * @param marcBibIds list of invalid marc bib ids
+   * @param tenantId tenant id
+   * @return future with list of invalid marc bib ids
+   */
+  Future<MarcBibCollection> verifyMarcBibRecords(List<String> marcBibIds, String tenantId);
```

Comment:
I think that `MarcBibCollection`naming is confusing because the only that we return is IDs of MARC_BIB records that were not found in the database.